docs/team/iaso1774.md

```diff
@@ -0,0 +1,57 @@
+# Ian Soukup - Project Portfolio Page
+
+## Overview
+`Manga Tantou` is a desktop Command Line Interface (CLI) program designed to help manga editors manage their business.
+The manga industry is known for being very tough to manage, with many deadlines, sales, and pieces of work to manage.
+Our program aims to be a catch-all tool for these tasks so a manga business can keep running efficiently and accurately.
+
+## Summary of Contributions
+
+### Code Contribution
+Refer to this [link](https://nus-cs2113-ay2425s1.github.io/tp-dashboard/?search=t10-3&sort=groupTitle%20dsc&sortWithin=title&since=2024-09-20&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=iaso1774&tabRepo=AY2425S1-CS2113-T10-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
+to view my contributions to the project.
+
+### New Feature: `Schedule`
+This feature allows deadlines to be quickly and reliably added to mangas.
+Through a simple Command Line input, editors can add or change the deadline associated with a manga.
+Additionally, through the view command, editors can easily check when deadlines are due for mangas.
+
+The manga industry is known for its demanding schedules. There have been many reports of authors being overworked and not given enough break time.
+We hope that by having a reliable and transparent deadline-assigning program in place, authors will be able to better plan around their busy schedules and manage their time more healthily.
+In addition, by giving the editor a wider view of all deadlines, they may be able to notice more easily if an author is overworked and give them less punishing deadlines.
+
+The `Schedule` command includes built-in checks to ensure that human error when inputting the command is caught.
+Checks include:
+* The author exists
+* The manga exists
+* The options are in the correct order
+* The correct number of arguments were provided
```

Comment:
These features are not part of the schedule command but the parser. In addition, our parser currently allows for dynamic positioning of arguments.

Reply:
Whoops, I forgot that we took that out. Let me fix that and merge.